src/screens/LoginPage/LoginPage.tsx

```diff
@@ -116,11 +116,82 @@ function LoginPage(): JSX.Element {
     }
   };
 
+  interface SignupFormData {
+    signfirstName: string;
+    signlastName: string;
+    signEmail: string;
+    signPassword: string;
+    cPassword: string;
+  }
+  interface ValidationResult {
```

Comment:
Why not use a type here instead of interfaces

Reply:
@Rumone is this resolved?